portfolio/src/main/webapp/index.html

```diff
@@ -48,13 +48,21 @@ <h2>Academic Achievements</h2>
     </ul>
 
     <p> View my transcript <a href="images/AcademicTranscript.pdf">here</a></p>
-    <button onclick="getFromServlet()">Click me!</button>
-    <div id="saying-container"></div>
     <br>
+    <h3>Give feedback!</h3>
     <form action="data" method="POST">
         <input type="text" name="user-comment">
         <input type="submit"/>
     </form>
+    <br>
+    <form>
+    <p> Number of comments to show</p>
+    <input type = "number" name="load-comments" value ="1" max="10"> 
+    <input type = "submit">
+    </form>
```

Comment:
This works, but it requires a full page refresh to update the comments (and a button press from the user).  A less expensive method would be to add an onchange event handler to the number input element.